### ISSUE TITLE
Bulk Upload

### ISSUE THREAD

**Issue by mnights101** [BNBFuse/milestones#12]
Manik, We are ready to bulk upload. Please let me know when the image URL script is ready, so she can proceed.

**Comment by mindrootstech** [comment]
image URL script?
Just send me your final CSV sheet in the format you sent on upwork.
I will get all the properties uploaded to the server.
Only thing i will need is the username under which all the properties should go.

**Comment by mnights101** [comment]
> Image Url  script?
Yes, The script we discussed, which would fetch image URL from source and upload it to s3 bucket.

[ha.zip](https://github.com/BNBFuse/milestones/files/790143/ha.zip)

**Comment by mnights101** [comment]
user name: concierge@bnbfusion.com
pass: mia209gt

**Comment by mindrootstech** [comment]
That would be a part of the overall data import script which we will do.

Cost will be $180
Please approve

**Comment by mnights101** [comment]
Actually I am paying script uploader to upload, please quote only script. I will see if bulk uploader would like to offer me a discount and cover your fee to have you upload.

**Comment by mindrootstech** [comment]
The script will be an auto-run script and there is actually no cost to run the script.
The cost is just to develop it..
Actually, you wont need anyone to run it.. You have given me the CSV. once the script is complete, we will just run it and all data will be imported.

**Comment by mnights101** [comment]
Ok, I have spoken with bulk uploader. Please proceed to create bulk upload script for 185 USD. Thanks.

**Comment by mnights101** [comment]
catch all email has been setup on inbox for bnbfusion you may now use any email @bnbfusion to setup fake profiles.

**Comment by mnights101** [comment]
as requested 100 fake profiles sent via upwork.

**Comment by mindrootstech** [comment]
Marco,

We imported a few properties form your final sheet but had to delete them as there are issues in the sheet.

1. The description has a lot of junk characters. Eg, this is the apartment features in first listing:
"Apartment Features
â€¢ 2 + 1 Apartment â€¢ 24 Hours Hot Water
â€¢ 85 m2 â€¢ Wifi / Internet
â€¢ 2 bedroom â€¢ LCD TV
â€¢ 4 person capacity â€¢ Satellite Broadcast
â€¢ Full furniture â€¢ Air Conditioning
â€¢ Appliances â€¢ Natural gas boiler
â€¢ American Kitchen â€¢ Laminate
â€¢ American â€¢ Modern Door Decoration
â€¢ Stylish Accessories â€¢ Clean Towels & sheets
â€¢ Steel Door â€¢ Sound & Thermal Insulation
â€¢ Security Camera â€¢ LED Lighting"


2. The images provided within each listing in CSV are multiple but all the links lead to the same image in different sizes: Eg, images in first listing. All of these are same image in different sizes:
https://imagesus-ssl.homeaway.com/mda01/371649f5-cb39-400f-9f2c-8fc35aa92a35.1.4,https://imagesus-ssl.homeaway.com/mda01/371649f5-cb39-400f-9f2c-8fc35aa92a35.1.8,https://imagesus-ssl.homeaway.com/mda01/371649f5-cb39-400f-9f2c-8fc35aa92a35.1.5,https://imagesus-ssl.homeaway.com/mda01/371649f5-cb39-400f-9f2c-8fc35aa92a35.1.3,https://imagesus-ssl.homeaway.com/mda01/371649f5-cb39-400f-9f2c-8fc35aa92a35.1.7,https://imagesus-ssl.homeaway.com/mda01/371649f5-cb39-400f-9f2c-8fc35aa92a35.1.9,https://imagesus-ssl.homeaway.com/mda01/371649f5-cb39-400f-9f2c-8fc35aa92a35.1.10,https://imagesus-ssl.homeaway.com/mda01/371649f5-cb39-400f-9f2c-8fc35aa92a35.1.6,https://imagesus-ssl.homeaway.com/mda01/371649f5-cb39-400f-9f2c-8fc35aa92a35.1.1,https://imagesus-ssl.homeaway.com/mda01/371649f5-cb39-400f-9f2c-8fc35aa92a35.1.2


3. Common, Additional Amenities and Special features:
Our database has a list of around 30 amenities (which you can see when you create a new listing) but our sheet has much more. We can import them as it is but each amenity in our system is linked with an icon. If we import the amenities in the sheet as it is, many wont have icons. I was thinking of just importing amenities which are present in our database and skipping the others.
What do you suggest here?

**Comment by mnights101** [comment]
Ok, I have notified the scraper, please stand by as we correct

**Comment by mnights101** [comment]
Corrected files sent via upwork. Please let me know if changes are satisfactory. also let me know hoe many pictures on average per property.

**Comment by mnights101** [comment]
sent another corrected file, she says images are now larger, please verify. as for  the amenities, I want to replace & match our amenities with homeaways amenities. I will create a new issue for this task.

**Comment by mnights101** [comment]
Manik, I was checking the bulk upload today, and I clicked on one of the reviews, it said user not found, I thought we where going to program script to pull reviews from fake profiles.

**Comment by mindrootstech** [comment]
Hi Marco,

The entire bulk upload is complete.
All properties and their images are now on bnbfusion.

Total cost: $450

**Comment by mnights101** [comment]
Manik, the task was agreed on 180, I did hear you mention it would be a bit more, however I discounted 180 from the scrapers fee to cover the work she was supposed to do, then you tell me it would be more after I release payment to her. I did not think it would be this much more, please adjust to price to a more reasonable amount.

**Comment by mindrootstech** [comment]
It genuinely took us that much time to process the .1 million records.
Anyways, i agree that i had quoted $180 and told you hat it would cost you a bit more.

 will take the hit here. Please approve $250 for this task

**Comment by mnights101** [comment]
paid via upwork
